cedar-policy-validator/src/str_checks.rs

```diff
@@ -232,7 +232,7 @@ mod test {
         );
         assert_eq!(
             format!("{warning}"),
-            "warning: string \"*_һello\" contains mixed scripts in policy with id test"
+            "warning: string `\"*_һello\"` contains mixed scripts in policy with id test"
```

Comment:
don't know where this error is generated so commenting on the test.. policy id 'test' should be surrounded with backticks.